Sources/DockProgress/DockProgress.swift

```diff
@@ -250,3 +264,47 @@ public enum DockProgress {
 		}
 	}
 }
+
+private typealias DisplayLinkObserverCallback = (DisplayLinkObserver, Double) -> Void;
```

Comment:
I would just pass the minimum needed, the stop method, not the whole instance.

```suggestion
private typealias DisplayLinkObserverCallback = (Double, () -> Void) -> Void
```